internal/libs/patchstruct/patchstruct_test.go

```diff
@@ -155,12 +155,12 @@ func TestPatchBytesErr(t *testing.T) {
 	t.Run("dst", func(t *testing.T) {
 		require := require.New(t)
 		_, err := PatchBytes([]byte("foo"), nil)
-		require.EqualError(err, "error reading destination data: proto: cannot parse invalid wire-format data")
+		require.Contains(err.Error(), "cannot parse invalid wire-format data")
```

Comment:
Merge plugin-hostcatalogs, this bug should be fixed in there.

Reply:
👍